bob-common/src/node.rs

```diff
@@ -39,8 +38,7 @@ pub struct Disk {
 impl Node {
     pub async fn new(name: String, address: &str, index: u16) -> Self {
         error!("address: [{}]", address);
-        let mut address = lookup_host(address).await.expect("DNS resolution failed");
-        let address = address.next().expect("address is empty");
+        let address = address.to_string();
```

Comment:
Pass address as `String` already because you make it owned here anyway

Reply:
Fixed